account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -26,7 +26,6 @@ class AccountPaymentLineCreate(models.TransientModel):
         selection=[("posted", "All Posted Entries"), ("all", "All Entries")],
         string="Target Moves",
     )
-    allow_blocked = fields.Boolean(string="Allow Litigation Move Lines")
```

Comment:
Why removing this? It's useful.

Reply:
Because it is now a property of the payment mode, not a question you ask to the use in this wizard. The user can use the button on the invoice to push it on the payment order and skip the wizard.